Modules/CalcOffence-3_0.lua

```diff
@@ -1172,13 +1178,82 @@ function calcs.offence(env, actor, activeSkill)
 		output.FistOfWarHitMultiplier = skillModList:Sum("BASE", cfg, "FistOfWarHitMultiplier") / 100
 		output.FistOfWarAilmentMultiplier = 1 + skillModList:Sum("BASE", cfg, "FistOfWarAilmentMultiplier") / 100
 		if output.FistOfWarCooldown ~= 0 then
-			output.FistOfWarHitEffect = 1 + output.FistOfWarHitMultiplier / (output.FistOfWarCooldown * output.Speed)
-			output.FistOfWarAilmentEffect = 1 + output.FistOfWarAilmentMultiplier / (output.FistOfWarCooldown * output.Speed)
+			output.FistOfWarHitEffect = 1 + output.FistOfWarHitMultiplier / m_max(output.FistOfWarCooldown * output.Speed, 1.0)
+			output.FistOfWarAilmentEffect = 1 + output.FistOfWarAilmentMultiplier / m_max(output.FistOfWarCooldown * output.Speed, 1.0)
 		else
 			output.FistOfWarHitEffect = 1
 			output.FistOfWarAilmentEffect = 1
 		end
 
+		-- Exerted Attack members
+		local exertedUptime = 0
+
+		-- Seismic Cry only Exerts Slam Skills
+		output.SeismicHitEffect = 1
+		if activeSkill.skillTypes[SkillType.SlamSkill] then
+			local numSeismicExerts = env.modDB:Sum("BASE", nil, "NumSeismicExerts") or 0
+			if numSeismicExerts > 0 and not skillFlags.warcry then
+				local moreDmgAndAoEPerExert = env.modDB:Sum("BASE", cfg, "SeismicMoreDmgPerExert") / 100
+				for index, value in ipairs(actor.activeSkillList) do
+					if value.activeEffect.gemData.name == "Seismic Cry" then
+						-- recursively calculate the values for Seismic Cry
+						-- only actual Cooldown and WarcryCastTime are returned
+						local seismicCryData = calcs.offence(env, actor, actor.activeSkillList[index], true)
+
+						output.SeismicCryCooldown = seismicCryData.Cooldown
+						output.SeismicCryCastTime = seismicCryData.WarcryCastTime
+						break
+					end
+				end
+				output.SeismicDmgImpact = 0
+				for i = 1, numSeismicExerts do
+					output.SeismicDmgImpact = output.SeismicDmgImpact + (i * moreDmgAndAoEPerExert)
+				end
+				output.SeismicAvgDmg = output.SeismicDmgImpact / numSeismicExerts
+				-- calculate ratio of uptime versus downtime (including opportunity cost of casting the warcry)
+				output.SeismicUpTimeRatio = m_min((numSeismicExerts / output.Speed) / (output.SeismicCryCooldown + output.SeismicCryCastTime), 1.0)
+				exertedUptime = m_max(exertedUptime, output.SeismicUpTimeRatio)
+				output.SeismicHitEffect = 1 + (output.SeismicAvgDmg * output.SeismicUpTimeRatio)
+				skillModList:NewMod("AreaOfEffect", "MORE", m_floor(output.SeismicAvgDmg * output.SeismicUpTimeRatio * 100), "Avg Seismic Exert AoE")
+			end
```

Comment:
You forgot to add the reduced stun threshold that Seismic Cry grants. Its base effect is based on Warcry Power and is scaled by buff effect; see my other comment near Intimidating Cry for a couple more details on that.